modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -134,7 +134,19 @@ Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path, uint32_
 
 	// Get global paths for source and sink.
 	// Escape paths to be valid Python strings to embed in the script.
-	const String source_global = ProjectSettings::get_singleton()->globalize_path(p_path).c_escape();
+	String source_global = ProjectSettings::get_singleton()->globalize_path(p_path)
```

Comment:
You forgot a `;`, so this doesn't even compile.

Reply:
Fixed!